### ISSUE TITLE
Including DataTables in pretty-plugin terminal output

### ISSUE THREAD

**Issue by artysidorenko** [cucumber/cucumber-jvm#2320]
**Is your feature request related to a problem? Please describe.**
Currently when you run scenarios with data tables, the table input is not included in the terminal output.

**Describe the solution you'd like**
Would like to include these tables in the output. I've got a [forked branch that shows a sample of the required changes I think](https://github.com/cucumber/cucumber-jvm/compare/main...artysidorenko:print-data-tables), however wasn't sure if it's something you would like to have as part of the existing plugin, switched on by default, or if these tables were originally excluded by design, and it's better to have it as some separate option if at all. Let me know what you think.

**Describe alternatives you've considered**
Was manually printing the tables via the step definitions, however the order of the items in the output was confusing.

**Comment by mpkorstanje** [comment]
This is a good suggestion!

To implement this the [`TablePrinter`](https://github.com/cucumber/common/blob/main/datatable/java/datatable/src/main/java/io/cucumber/datatable/TablePrinter.java) could be reused.

Though [it currently prints everything at a fixed indent ](https://github.com/cucumber/common/blob/main/datatable/java/datatable/src/main/java/io/cucumber/datatable/TablePrinter.java#L27) and [escapes the cell content](https://github.com/cucumber/common/blob/main/datatable/java/datatable/src/main/java/io/cucumber/datatable/TablePrinter.java#L73). The latter isn't great for readability. So this could be made configurable. Something like:

```java
DataTablePrinter printer = DataTablePrinter.builder()
    .indent("      ")
    .escapeCells(false)
    .build();

printer.print(table, out);
```

Additionally the current `TablePrinter` and would need a once over to be made part of the public API. Would you like to prepare the PRs for this?

**Comment by artysidorenko** [comment]
Sure, I'll put something together. Hadn't come across the cucumber/common repo yet but I'll familiarise myself and have a stab at it 👍

**Comment by mpkorstanje** [comment]
Fixed by #2330
